erdpy/proxy/core.py

```diff
@@ -120,3 +120,16 @@ def get_hyperblock(self, key) -> Any:
         response = do_get(url)
         response = response.get("hyperblock", {})
         return response
+
+    def send_transaction_and_wait_for_result(self, payload: Any) -> str:
+        url = f"{self.url}/transaction/send"
+        response = do_post(url, payload)
+        tx_hash = response.get("txHash")
+
+        while True:
```

Comment:
Although this works, if we send a wrong transaction the whole script will block. Maybe add a timeout value in seconds? and make this function return both str and error?

Reply:
Done.